features/webdriver-bidi.yml.dist

```diff
@@ -1,4 +1,290 @@
+
+  # baseline: false
+  # support:
+  #   chrome: "126"
+  #   chrome_android: "126"
+  #   edge: "126"
+  #   firefox: "129"
+  #   firefox_android: "129"
+  - webdriver.bidi.browser.close
+  - webdriver.bidi.browser.removeUserContext
+  - webdriver.bidi.browsingContext.close
+  - webdriver.bidi.browsingContext.handleUserPrompt
+
+  # baseline: false
+  # support:
+  #   chrome: "126"
+  #   chrome_android: "126"
+  #   edge: "126"
+  #   firefox: "136"
+  #   firefox_android: "136"
+  - webdriver.bidi.session.subscribe
+  - webdriver.bidi.session.unsubscribe
+
+  # baseline: false
+  # support:
+  #   chrome: "131"
+  #   chrome_android: "131"
+  #   edge: "131"
+  #   firefox: "135"
+  #   firefox_android: "135"
+  - webdriver.bidi.browsingContext.captureScreenshot
+
+  # baseline: false
+  # support:
+  #   chrome: "132"
+  #   chrome_android: "132"
+  #   edge: "132"
+  #   firefox: "130"
+  #   firefox_android: "130"
+  - webdriver.bidi.network.setCacheBehavior
+
+  # baseline: false
+  # support:
+  #   chrome: "126"
+  #   chrome_android: "126"
+  #   edge: "126"
+  - webdriver.bidi.browsingContext.contextCreated_event
+  - webdriver.bidi.browsingContext.contextDestroyed_event
+  - webdriver.bidi.browsingContext.create
+  - webdriver.bidi.browsingContext.locateNodes
+  - webdriver.bidi.browsingContext.navigate
+  - webdriver.bidi.browsingContext.navigationCommitted_event
+  - webdriver.bidi.browsingContext.navigationFailed_event
+  - webdriver.bidi.browsingContext.reload
+  - webdriver.bidi.browsingContext.setViewport
+  - webdriver.bidi.browsingContext.traverseHistory
+  - webdriver.bidi.browsingContext.userPromptClosed_event
+  - webdriver.bidi.browsingContext.userPromptOpened_event
+  - webdriver.bidi.input.setFiles
+  - webdriver.bidi.log.entryAdded_event
+  - webdriver.bidi.network.continueWithAuth
+  - webdriver.bidi.script.getRealms
+  - webdriver.bidi.script.realmCreated_event
+  - webdriver.bidi.session.new
+
+  # baseline: false
+  # support:
+  #   chrome: "133"
+  #   chrome_android: "133"
+  #   edge: "133"
+  - webdriver.bidi.network.fetchError_event
+
+  # baseline: false
+  # support:
+  #   firefox: "122"
+  #   firefox_android: "122"
+  - webdriver.bidi.network.authRequired_event
+
+  # baseline: false
+  # support:
+  #   firefox: "134"
+  #   firefox_android: "134"
+  - webdriver.bidi.browser.getClientWindows
+
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: false
+  # support: {}
+  - webdriver.bidi.browsingContext.print
+  - webdriver.bidi.network.addIntercept
+  - webdriver.bidi.network.beforeRequestSent_event
+  - webdriver.bidi.network.continueRequest
+  - webdriver.bidi.network.continueResponse
+  - webdriver.bidi.network.provideResponse
+  - webdriver.bidi.network.responseCompleted_event
+  - webdriver.bidi.network.responseStarted_event
+  - webdriver.bidi.session.end
+  - webdriver.bidi.webExtension
+  - webdriver.bidi.webExtension.install
+  - webdriver.bidi.webExtension.uninstall
```

Comment:
These keys are not supported anywhere. Good candidates to move elsewhere if they belong to a separate feature / later addition.

Reply:
The `webExtension` module is a later addition to the spec. I think we should pin support to the earliest parts of BiDi, and treat this as a later addition.